autodoc.pl

```diff
@@ -498,13 +498,28 @@ ($$)
 
     my $file_is_C = $file =~ / \. [ch] $ /x;
 
-    # Count lines easier
-    my $get_next_line = sub { $line_num++; return <$fh> };
+    # Count lines easier and handle apidoc continuation lines
+    my $get_next_line = sub {   my $contents = <$fh>;
```

Comment:
Can we wrap this after the `{` and indent the sub body the normal amount instead of half a screen?

Reply:
I was trying to preserve pre-existing style, but sure. Now done